_ide_helper.php

```diff
@@ -86,4 +90,71 @@ public function assertGraphQLValidationPasses(): self
     {
         return $this;
     }
+
+    /**
+     * Assert current user is authorized to join a subscription.
+     *
+     * @param  \PHPUnit\Framework\TestCase  $testClassInstance  usually $this when calling this from a test class
+     *
+     * @return $this
+     */
+    public function assertGraphQLSubscriptionAuthorized(TestCase $testClassInstance): self
+    {
+        return $this;
+    }
+
+    /**
+     * Assert current user is NOT authorized to join a subscription.
+     *
+     * @param  \PHPUnit\Framework\TestCase  $testClassInstance  usually $this when calling this from a test class
+     *
+     * @return $this
+     */
+    public function assertGraphQLSubscriptionNotAuthorized(TestCase $testClassInstance): self
+    {
+        return $this;
+    }
+
+    /**
+     * For cases where you need more control over your broadcast assertions.
+     *
+     * @see assertGraphQLBroadcasted
+     * @see assertGraphQLNotBroadcasted
+     *
+     * @return \Mockery\MockInterface&\Nuwave\Lighthouse\Subscriptions\Broadcasters\LogBroadcaster
```

Comment:
@spawnia phpstan gives this error:

```
126    Method Illuminate\Testing\TestResponse::graphQLSubscriptionMock() has invalid return type Nuwave\Lighthouse\Subscriptions\Broadcasters\LogBroadcaster.
```

Which is probably because the return types don't match we can't use:

```php
public function graphQLSubscriptionMock(): \Mockery\MockInterface&\Nuwave\Lighthouse\Subscriptions\Broadcasters\LogBroadcaster
```

Because `Intersection types are only allowed since PHP 8.1 ` and I think this project's target is 8.0 not 8.1 right?

Reply:
Target aside even doing that doesn't fix the phpstan error